src/cedar.js

```diff
@@ -357,6 +358,9 @@ Cedar.prototype.show = function(options){
     if(options.autolabels !== undefined && options.autolabels !== null){
       this.autolabels = options.autolabels;
     }
+    // check if truncate label length has been passed in
+    this.truncateLabelLength = options.truncateLabelLength || 20;
```

Comment:
Except it's tied to length / how far the labels and title are from each other.  `undefined` makes all labels `...`. `20` because that gives a good default appearance. If someone wants it longer they can set it longer. I believe VL is doing 20 - 25 for their defaults.